client/source/DelphiLint.ToolFrame.dfm

```diff
@@ -247,11 +247,75 @@ object LintToolFrame: TLintToolFrame
       MarqueeInterval = 30
       TabOrder = 0
     end
+    object ErrorButtonPanel: TPanel
+      Left = 576
+      Top = 0
+      Width = 132
+      Height = 27
+      Align = alRight
+      BevelOuter = bvNone
+      Color = clRed
+      ParentBackground = False
+      TabOrder = 1
+      StyleElements = [seFont, seBorder]
+      object ErrorButton: TSpeedButton
+        Left = 0
+        Top = 0
+        Width = 132
+        Height = 27
+        Align = alClient
+        Caption = '2 errors'
```

Comment:
Why is the caption "2 errors" by default?

Reply:
It's just something to look at at design-time. Do you think I should make it something else?

Reply:
Nah, that seems fine.